e2e/provider/tests/meta.spec.ts

```diff
@@ -0,0 +1,6 @@
+import { expect, test } from "@playwright/test";
+
+test.only("Story meta is available in provider", async ({ page }) => {
```

Comment:
leftover `.only`?

Reply:
Hum, I guess I can only blame that on a lazy copy paste 😁 Removed the .only now